assets/js/modules/tagmanager/datastore/__fixtures__/index.js

```diff
@@ -16,6 +16,21 @@
  * limitations under the License.
  */
 
+/**
+ * Internal dependencies
+ */
+import ContainersAMPOnly from './get-containers--amp';
+import ContainersWebOnly from './get-containers--web';
+
 export { default as accounts } from './accounts';
 export { default as createContainer } from './create-container';
 export { default as liveContainerVersion } from './live-container-version.json';
```

Comment:
@tofumatt mentioned something similar. It works with/without just like `.js` files and I don't think we've ever made a decision about this for json files. I agree they should at least all be consistent so that was a miss here for sure. Not sure if it adds any benefit including `.json` or not though. I'll update them to use `json` extensions.

Reply:
Updated! Now that it's changed I can see that VS Code offers inspection into the contents of the file where before it didn't, so this is better 👍

Reply:
For what it's worth: as a general rule, I like using filename extensions for anything that _isn't_ code/JS. I suppose it's a bit arbitrary, but I find it helps clarify what a file is—no extension implies JS code, anything else you can infer from the filename.

Nice that it has a bonus of improving VSCode inspection 🙂